modules/core/src/core-layers/text-layer/font-atlas.js

```diff
@@ -36,33 +36,39 @@ export function makeFontAtlas(
   // measure texts
   let row = 0;
   let x = 0;
+  let fontHeight = 0;
   const mapping = {};
 
   Array.from(characterSet).forEach(char => {
-    const {width} = ctx.measureText(char);
+    const {width, fontBoundingBoxDescent} = ctx.measureText(char);
```

Comment:
Maybe add a comment about "advanced text metrics" only being available on Chrome? https://developer.mozilla.org/en-US/docs/Web/API/TextMetrics

Worth doing any tests on non-Chrome browsers? If still an issue, how do other apps work around it?